source/fe/ast/parser.d

```diff
@@ -359,7 +367,15 @@ struct Parser
 				import std.algorithm.iteration : filter;
 				uint syscallNumber = value.filter!(c => c != '_').to!uint;
 				nextToken; // skip integer
-				attribute = make!BuiltinAttribNode(start, BuiltinAttribSubType.extern_syscall, syscallNumber);
+				attribute = make!BuiltinAttribNode(start, BuiltinAttribSubType.extern_syscall, Identifier(syscallNumber));
+				break;
+			case CommonIds.id_module.index:
+				expectAndConsume(TokenType.COMMA, "@extern(module");
+				expect(TT.STRING_LITERAL, "@extern(module,");
+				string value = cast(string)context.getTokenString(tok.index);
+				nextToken; // skip lib name
+				Identifier moduleId = context.idMap.getOrRegNoDup(context, value);
+				attribute = make!BuiltinAttribNode(start, BuiltinAttribSubType.extern_module, moduleId);
```

Comment:
What I meant is that `Identifier.index` can be stored in `uint data` of `BuiltinAttribNode`.